File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/note/NoteMvpView.java
package com.mifos.mifosxdroid.online.note;

import com.mifos.mifosxdroid.base.MvpView;
import com.mifos.objects.noncore.Note;

import java.util.List;

/**
 * Created by rahul on 4/3/17.
 */
public interface NoteMvpView extends MvpView {

    void showUserInterface();

    void showNote(List<Note> note);

    void showEmptyNotes();

    void showResetVisibility();

    void showError(int message);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/note/NoteMvpView.kt
package com.mifos.mifosxdroid.online.note

import com.mifos.mifosxdroid.base.MvpView
import com.mifos.objects.noncore.Note

/**
 * Created by rahul on 4/3/17.
 */
interface NoteMvpView : MvpView {
    fun showUserInterface()
    fun showNote(note: List<Note>)
    fun showEmptyNotes()
    fun showResetVisibility()
    fun showError(message: Int)
}